instrumentctl/g9_driver.py

```diff
@@ -86,12 +82,20 @@ def _setup_serial(self, port, baudrate, timeout):
                     bytesize=serial.EIGHTBITS,  
                     timeout=timeout  
                     ) 
+                
+                self._lock = threading.Lock()
+                self._response_queue = queue.Queue(maxsize=1)
+                self._running = True
+                self._thread = threading.Thread(target=self._communication_thread, daemon=True)
+                self._thread.start()
```

Comment:
@mark11778 these changes affect the g9_driver.py but aren't reflected in the G9 driver PR. 

Can you move these to [PR#16](https://github.com/uw-loci/EBEAM_dashboard/pull/16) and paste in a fresh log file to the comments (without committing the logfile) so that I can merge it?

Reply:
Ope, sorry I missed this I will make a branch now